README.md

```diff
@@ -66,10 +66,8 @@ const response = await fetch(`${baseUrl}/guests/1`, { method: 'DELETE' });
 const deletedGuest = await response.json();
 ```
 
-## Deploying to Heroku
```

Comment:
let's comment this out (HTML comments) instead of removing it completely, in case Heroku student plans will work (these will be announced near end of September 2022)

Reply:
93b012cfd8f82e8618262928c0d862cbda054940

Reply:
One further change here: let's add the start and end of the HTML comments on new lines (with an empty line in between), so that the diff does not touch the existing lines of the Heroku section

Reply:
db78899